src/api.py

```diff
@@ -196,19 +198,39 @@ def recitation(self):
                 else:
                     raise EditorNotFound()
             except EditorNotFound:
-                print("Text editor not found; set the 'EDITOR'" +
+                if platform.system() == "Windows":
+                    editor = "notepad"
+                    windowsfilename = f"{self.target.reference}"
+                    windowsfilename = windowsfilename.replace(":","-")
+                else:
+                    print("Text editor not found; set the 'EDITOR'" +
                       "environmental variable and try again")
-                exit()
+                    exit()
+
+        if platform.system() == "Windows":
+            self.filename = self.cache_path / windowsfilename
+        else:
+            self.filename = self.cache_path / f"{self.target.reference}"
 
-        self.filename = self.cache_path / f"{self.target.reference}"
-        subprocess.run([editor, self.filename])
+        if os.path.isfile(self.filename) == False:
+            with open(self.filename, "w") as file:
+                subprocess.run([editor, self.filename])
```

Comment:
This strikes me as dangerous because the file is open in both Python and in the subprocess at the same time! Perhaps we could create a blank file first (if that is why this is here) with touch:
```suggestion
            self.filename.touch(exist_ok=True)
            subprocess.run([editor, self.filename])
```